pkg/cmd/cli/pair.go

```diff
@@ -251,3 +270,11 @@ func (c *pairConfig) match(ctx context.Context, outputPath string, publisherPAIR
 
 	return nil
 }
+
+func readersFromReadClosers(rs []io.ReadCloser) []io.Reader {
+	readers := make([]io.Reader, len(rs))
+	for i, r := range rs {
+		readers[i] = r
+	}
+	return readers
+}
```

Comment:
still needed?

Reply:
It is actually still needed, we still perform matches, but not as a standalone command

Reply:
ah this was from the removed files